Launch file documentation for jupyterlab-python (#11162)
PrairieLearn/PrairieLearn: workspaces/jupyterlab-python/README.md

# workspace-jupyterlab-python

> To automatically open a file on launch of this workspace, please use the `LAUNCH_FILE_NAME` environment variable in your `info.json` as shown in the example below.
> 
> ```
> "environment": {
>     "LAUNCH_FILE_NAME": "some_file.ext"
> }
> ```
> 
> Replace `some_file.ext` with your file. It is expected that this file exists inside of the `/workspace` directory of the question. Please see the [workspaces documentation](https://prairielearn.readthedocs.io/en/latest/workspaces/) for more details on environment variable usage.

Reviewer: To me, this suggests that it should go in the top level of the JSON file. Let's clarify that it must go under `workspaceOptions`, as in this example:

https://github.com/PrairieLearn/PrairieLearn/blob/9d539e13cf80051653d715dd2574c960fde816df/testCourse/questions/workspace/info.json#L12

A few more opinionated nits (e.g. we generally don't need "please" in docs) too.

````suggestion
To automatically open a file when the workspace is started, set the `LAUNCH_FILE_NAME` environment variable in your question's `info.json`:

```json
{
  "workspaceOptions": {
    "image": "prairielearn/workspace-jupyterlab-python",
    "environment": {
      "LAUNCH_FILE_NAME": "some_file.ext"
    }
  }
}
```

Replace `some_file.ext` with your file. It is expected that this file exists inside of the `/workspace` directory of the question. See the [workspaces documentation](https://prairielearn.readthedocs.io/en/latest/workspaces/) for more details on environment variable usage.
````
